menu/menu.yaml

```diff
@@ -258,6 +258,8 @@
         Title: Troubleshooting
       - Url: nservicebus/gateway/scale-out
         Title: Scale-Out
+    - Url: nservicebus/wormhole
+      Title: Worm Hole
```

Comment:
~~Worm Hole~~ Wormhole

Shouldn't this be "Wormhole Gateway" though? Sitting between "Gateway" and "Outbox" just the word "Wormhole" doesn't give a lot of context.